src/bot.py

```diff
@@ -36,15 +37,9 @@
     ]
 )
 
-mininterval = config['mininterval']
-maxinterval = config['maxinterval']
-
-cooldown = randint(mininterval, maxinterval) * 24 * 60 * 60
-# converts the random value from days to seconds
-
-limit = config['lines'] * config['limit_per_line']
-
+emojis = ["🌻", "🌱", "🌸", "🌷", "💮", "🌺", "🌹", "🌼", "🌿", "🌿", "🌷"]
 
+config_path = "src/config.txt"

```

Comment:
Hm, what if the end user is running the bot manually, while having `cd'd` into the `src` directory itself?

Reply:
Simply wouldn't work. I just followed the original project structure.

Reply:
Oh yeah, you're right about the fact that this is how I did it in the first place.

Please excuse me for those questions, it's just that although I've turned this bot into an educational project a while ago, the fact still is that I haven't touched it myself in a while and have forgotten a bunch of things revolving around it. 😬